src/dev/run_check_core_api_changes.ts

```diff
@@ -0,0 +1,187 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { Extractor, IExtractorConfig } from '@microsoft/api-extractor';
+import chalk from 'chalk';
+import dedent from 'dedent';
+import execa from 'execa';
+import getopts from 'getopts';
+import { run } from './run';
+
+const apiExtractorConfig: IExtractorConfig = {
```

Comment:
Since it's used for generating the documentation we want others to use, I think it might be better to be explicit about which API's are public, beta or private.